ci/validate_wheel.sh

```diff
@@ -12,6 +12,7 @@ rapids-logger "validate packages with 'pydistcheck'"
 
 pydistcheck \
     --inspect \
+    --output-file-size-precision=4 \
```

Comment:
This appeared to be [supported upstream]( https://github.com/jameslamb/pydistcheck/blob/4afb495763938471721f2c1fa7c1b86ea02fc339/src/pydistcheck/cli.py#L164-L172 )


However am seeing [the following error on CI]( https://github.com/rapidsai/cudf/actions/runs/14921772918/job/41918447206#step:11:5808 ):

```
Usage: pydistcheck [OPTIONS] [FILEPATHS]...
Try 'pydistcheck --help' for help.

Error: No such option: --output-file-size-precision
```